src/Components/Facility/PatientNotesSlideover.tsx

```diff
@@ -0,0 +1,145 @@
+import { useState, useEffect, Dispatch, SetStateAction } from "react";
+import { getPatient, addPatientNote } from "../../Redux/actions";
+import * as Notification from "../../Utils/Notifications.js";
+import { useDispatch } from "react-redux";
+import PatientNotesList from "./PatientNotesList";
+import { NonReadOnlyUsers } from "../../Utils/AuthorizeFor";
+import CareIcon from "../../CAREUI/icons/CareIcon";
+import { classNames } from "../../Utils/utils";
+import TextFormField from "../Form/FormFields/TextFormField";
+import ButtonV2 from "../Common/components/ButtonV2";
+import { make as Link } from "../Common/components/Link.bs";
+
+interface PatientNotesProps {
+  patientId: string;
+  facilityId: string;
+  setShowPatientNotesPopup: Dispatch<SetStateAction<boolean>>;
+}
+
+export default function PatientNotesSlideover(props: PatientNotesProps) {
+  const [show, setShow] = useState(false);
+  const [patientActive, setPatientActive] = useState(true);
+  const [noteField, setNoteField] = useState("");
+  const [reload, setReload] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const { facilityId, patientId, setShowPatientNotesPopup } = props;
+
+  const onAddNote = () => {
+    const payload = {
+      note: noteField,
+    };
+    if (!/\S+/.test(noteField)) {
+      Notification.Error({
+        msg: "Note Should Contain At Least 1 Character",
+      });
+      return;
+    }
+    dispatch(addPatientNote(patientId, payload)).then(() => {
+      Notification.Success({ msg: "Note added successfully" });
+      setNoteField("");
+      setReload(!reload);
+    });
+  };
+
+  useEffect(() => {
+    async function fetchPatientName() {
+      if (patientId) {
+        const res = await dispatch(getPatient({ id: patientId }));
+        if (res.data) {
+          setPatientActive(res.data.is_active);
+        }
+      }
+    }
+    fetchPatientName();
+  }, [dispatch, patientId]);
+
+  const notesActionIcons = (
+    <div className="flex gap-1">
+      {show && (
+        <Link
+          className={classNames(
+            "flex h-8 w-8 cursor-pointer items-center justify-center rounded bg-primary-800 text-gray-100 text-opacity-70 hover:bg-primary-700 hover:text-opacity-100"
+          )}
```

Comment:
You need not use `classNames` utility if u are not using conditional classnames.

```suggestion
          className="flex h-8 w-8 cursor-pointer items-center justify-center rounded bg-primary-800 text-gray-100 text-opacity-70 hover:bg-primary-700 hover:text-opacity-100"
```